distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/NsiliClient.java

```diff
@@ -755,7 +748,7 @@ public String getIorTextFile(String iorURL) throws Exception {
         }
 
         if (StringUtils.isNotBlank(myString)) {
-            System.out.println("Successfully Downloaded IOR File From Server.\n");
+            LOGGER.info("Successfully Downloaded IOR File From Server.");
             return myString;
         }
 
```

Comment:
Outside the scope of this PR:  `receiving` is misspelled in `"Error recieving IOR File"`